ta4j-core/src/main/java/org/ta4j/core/BacktestingResult.java

```diff
@@ -0,0 +1,93 @@
+package org.ta4j.core;
+
+import org.ta4j.core.analysis.criteria.*;
+import org.ta4j.core.num.Num;
+
+/**
+ * This class contains the result of a backtested strategy
+ */
+public class BacktestingResult implements Comparable<BacktestingResult> {
+
+    private Strategy strategy;
+    private TradingRecord tradingRecord;
+    private Num totalProfitLoss;
+    private Num totalProfitLossPercentage;
+    private Num totalProfit;
+    private Num totalLoss;
+    private Num profitTradeCount;
```

Comment:
I think Num is good choice for double/decimal values, but counts should be integer values

Reply:
Num is from the AbstractAnalysisCriterion class. We probebly should make one abstract class that results in a int.